Multisig/Logic/AuthenticationController.swift

```diff
@@ -112,7 +112,7 @@ class AuthenticationController {
     }
 
     private func derivedKey(from plaintext: String) -> String {
-        let salt = "Safe Multisig Passcode Salt"
+        let salt = "Gnosis Safe Multisig Passcode Salt"
```

Comment:
oh shit. @MouazAlzahabi please add a comment before this line that this must be kept the same so that the passcodes can be decoded from earlier versions